pandas/tests/test_strings.py

```diff
@@ -1906,45 +1906,6 @@ def test_split_no_pat_with_nonzero_n(self):
 
     def test_split_to_dataframe(self):
         s = Series(['nosplit', 'alsonosplit'])
-
-        with tm.assert_produces_warning(FutureWarning):
```

Comment:
They are duplicates essentially with the tests that follow, which is why I removed them.